tests/pdctl/helper.go

```diff
@@ -130,3 +130,24 @@ func MustPutRegion(re *require.Assertions, cluster *tests.TestCluster, regionID,
 	re.NoError(err)
 	return r
 }
+
+// MustReportBuckets is used for test purpose.
+func MustReportBuckets(re *require.Assertions, cluster *tests.TestCluster, regionID uint64, start, end []byte, loads []uint64) *metapb.Buckets {
```

Comment:
I think it's better to pass `BucketStats`
```suggestion
func MustReportBuckets(re *require.Assertions, cluster *tests.TestCluster, regionID uint64, start, end []byte, stats *metapb.BucketStats) *metapb.Buckets {
```